packages/frontend/src/components/RefreshServer/RefreshStepsModal/index.tsx

```diff
@@ -0,0 +1,165 @@
+import { Classes, Drawer, Icon, IconName, Position } from '@blueprintjs/core';
+import React, { Dispatch, FC, SetStateAction } from 'react';
+import {
+    ErrorMessageWrapper,
+    RefreshStepsHeadingWrapper,
+    RefreshStepsTitle,
+    Step,
+    StepIcon,
+    StepName,
+    StepsCompletionOverview,
+    StepStatus,
+    StepStatusWrapper,
+    StepsWrapper,
+} from './RefreshStepsModal.style';
+
+const refreshStepsMockup = {
+    jobStatus: 'RUNNING', // refresh status ('DONE' | 'RUNNING' | 'ERROR')
+    jobResults: '/projects/{projectUuid}/explores',
+    steps: [
+        {
+            name: 'Clone dbt project from github', // step name
+            stepStatus: 'DONE', // step status ('DONE' | 'RUNNING' | 'ERROR' | 'PENDING' | 'SKIPPED')
+            runningTime: '00:00:31', // time it has run for in mins/secs
+        },
+        {
+            name: 'Install dbt project dependencies ', // step name
+            stepStatus: 'RUNNING', // step status ('DONE' | 'RUNNING' | 'ERROR' | 'PENDING' | 'SKIPPED')
+            runningTime: '00:00:31', // time it has run for in mins/secs
+        },
+        {
+            name: 'Compile dbt project', // step name
+            stepStatus: 'ERROR', // step status ('DONE' | 'RUNNING' | 'ERROR' | 'PENDING' | 'SKIPPED')
+            error: {
+                name: 'Name n_stores not found inside',
+                message: 'campaign_all_brand-hist at [2:33]',
+            },
+            runningTime: '00:00:31', // time it has run for in mins/secs
+        },
+        {
+            name: 'Get latest data warehouse schema', // step name
+            stepStatus: 'PENDING', // step status ('DONE' | 'RUNNING' | 'ERROR' | 'PENDING' | 'SKIPPED')
+            runningTime: '00:00:31', // time it has run for in mins/secs
+        },
+        {
+            name: 'Compile metrics and dimensions', // step name
+            stepStatus: 'PENDING', // step status ('DONE' | 'RUNNING' | 'ERROR' | 'PENDING' | 'SKIPPED')
+            runningTime: '00:00:31', // time it has run for in mins/secs
```

Comment:
Looks like the BE is going to return a created + updated time values. So we will need to make the calculations and formatting here.